pkg/collect/host_network_test.go

```diff
@@ -44,8 +44,13 @@ func Test_isValidLoadBalancerAddress(t *testing.T) {
 			want: false,
 		},
 		{
-			name: "Too many characters",
+			name: "Too many characters less than 255",
```

Comment:
what is this testing for?  The name needs to be really clear.

Reply:
good catch. I have updated those test names